### PR TITLE
Added a tooltip for the Ore Gen coordinate system

### LINKED ISSUE #41: Label oregen axis at points other than the very beginning and end
In its current state it's difficult to tell exactly where the dense part actually begins and ends.
Perhaps start at bestY, then label the endpoints of the line segment it lands on? (or keep it simple and label every 5 or 10)

### PR DESCRIPTION
**Pull request by FantasyTeddy**
I added a tooltip for the coordinate system in the ore gen GUI. It tells you the Y height (and the chance in %) of the current point you are hovering over.

**Example:**
hover

This possibly closes #41.

### CODE REVIEW

File: src/main/java/neresources/nei/NEIOreHandler.java [hunk 1]
```diff
@@ -109,6 +114,31 @@ public void drawExtras(int recipe)
...
+            if (relMouse.x > X_OFFSPRING && relMouse.x < X_OFFSPRING + X_AXIS_SIZE &&
+                relMouse.y > Y_OFFSPRING - Y_AXIS_SIZE && relMouse.y < Y_OFFSPRING)
+            {
+                CachedOre cachedOre = (CachedOre) arecipes.get(recipe);
+                float[] chances = cachedOre.oreMatchEntry.getChances();
+                double space = X_AXIS_SIZE / (chances.length * 1D);
+                // Calculate the hovered over y value
+                int yValue = (int) ((relMouse.x - X_OFFSPRING) / space);
+                if (yValue > 0 && yValue < chances.length)
+                    //TODO: The shift of one element here is due to some minor inaccuracy in the drawing function and could be avoided
```
Comment (way2muchnoise) [inline src/main/java/neresources/nei/NEIOreHandler.java]: Please use brackets when ifs are more then one like. When it's just one no brackets are fine but with comment it looks more clean with them, gives more overview

File: src/main/java/neresources/nei/NEIOreHandler.java [hunk 2]
```diff
@@ -109,6 +114,33 @@ public void drawExtras(int recipe)
     }
 
     @Override
+    public List<String> handleTooltip(GuiRecipe gui, List<String> currenttip, int recipe)
+    {
+        if (GuiContainerManager.shouldShowTooltip(gui) && currenttip.size() == 0)
+        {
+            Point offset = gui.getRecipePosition(recipe);
+            Point pos = getMousePosition();
```
Comment (way2muchnoise) [inline src/main/java/neresources/nei/NEIOreHandler.java]: I don't like static imports ^^ So could you make it GuiDraw.getMousePosition() ?

File: src/main/java/neresources/nei/NEIOreHandler.java (hunk 1 above)
Comment (FantasyTeddy) [inline src/main/java/neresources/nei/NEIOreHandler.java]: Okay, done in 6b4ebb8da41f460630d7fe6fc1d2442d348212be

File: src/main/java/neresources/nei/NEIOreHandler.java (hunk 2 above)
Comment (FantasyTeddy) [inline src/main/java/neresources/nei/NEIOreHandler.java]: Sure ;-)
Here: dc58d3742254fa1f8aeb663d8b0910e28e8601a9

**Comment by way2muchnoise** [comment]
Thx! btw if you ever want to bounce any ideas of @hilburn or me you can normally find us in #minechem on esper.net IRC

**Comment by FantasyTeddy** [comment]
Okay cool, I will eventually drop by :smile:

**Pull request merged**